deepmd/pt/model/model/sezm_model.py

```
@@ -1632,6 +1663,7 @@ def forward_lower(
         do_atomic_virial: bool = False,
         comm_dict: dict[str, torch.Tensor] | None = None,
         charge_spin: torch.Tensor | None = None,
+        extended_atype: Int[Tensor, "nf nall"] | None = None,
```

Ruff (0.15.18)

[error] Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)